modules/display.js

```diff
@@ -0,0 +1,15 @@
+const display = (players) => {
```

Comment:
[OPTIONAL] I recommend moving the `modules` folder to the `/src` folder so your project is well organized.